com.atlauncher.ATLauncher.desktop

```diff
@@ -0,0 +1,11 @@
+# Credits to github/nickavem for the file, Doomsdayrs for edits
```

Comment:
This file should be submitted upstream.

Reply:
So in https://github.com/ATLauncher/ATLauncher/tree/master/packaging/linux ?

Reply:
Reason for this is because the file is different